netket/abstract_driver.py

```diff
@@ -0,0 +1,281 @@
+import abc
+
+from netket._core import deprecated
+import netket as _nk
+
+from netket.logging import JsonLog as _JsonLog
+
+from netket.vmc_common import make_optimizer_fn, tree_map
+
+from tqdm import tqdm
+
+import warnings
+
+
+def obs_stat_to_dict(value):
```

Comment:
This should be internal, shouldn't it?
```suggestion
def _obs_stat_to_dict(value):
```